### PR TITLE
feat(EMS-608): Policy and exports - About goods or services - description field

### PR DESCRIPTION
**Pull request by ttbarnes**
This PR adds validation and data saving for the description field in the "About goods or services" page.

## Summary of changes

- Add `goodsOrServicesDescription` to the schema, DB dump, application types and GQL query.
- Add validation rules.
- Add POST controlelr with `mapAndSave` functionality.
- Add E2E tests.

### CODE REVIEW

File: e2e-tests/content-strings/error-messages.js [hunk 1]
```diff
@@ -141,6 +141,15 @@ export const ERROR_MESSAGES = {
           },
         },
       },
+      ABOUT_GOODS_OR_SERVICES: {
+        [FIELD_IDS.INSURANCE.POLICY_AND_EXPORTS.ABOUT_GOODS_OR_SERVICES.DESCRIPTION]: {
+          IS_EMPTY: 'Enter the goods or services you will supply to the buyer',
+          ABOVE_MAXIMUM: 'The goods or services you will supply to the buyer cannot be more than 1000 characters.',
```
Comment (Zainzzkk) [inline e2e-tests/content-strings/error-messages.js:147]: is there a reason this one has a full stop and the others don't? should we keep it standard for all with/without punctuation at the end unless its a question

File: src/ui/server/content-strings/error-messages.ts [hunk 2]
```diff
@@ -141,6 +141,15 @@ export const ERROR_MESSAGES = {
           },
         },
       },
+      ABOUT_GOODS_OR_SERVICES: {
+        [FIELD_IDS.INSURANCE.POLICY_AND_EXPORTS.ABOUT_GOODS_OR_SERVICES.DESCRIPTION]: {
+          IS_EMPTY: 'Enter the goods or services you will supply to the buyer',
+          ABOVE_MAXIMUM: 'The goods or services you will supply to the buyer cannot be more than 1000 characters.',
```
Comment (Zainzzkk) [inline src/ui/server/content-strings/error-messages.ts:147]: same as above about end punctuation

File: src/ui/server/controllers/insurance/policy-and-export/about-goods-or-services/index.ts [hunk 3]
```diff
@@ -65,3 +68,51 @@ export const get = (req: Request, res: Response) => {
...
+    // save the application
+    const saveResponse = await mapAndSave.policyAndExport(req.body, application);
+
+    if (!saveResponse) {
+      return res.redirect(ROUTES.PROBLEM_WITH_SERVICE);
+    }
+
+    return res.redirect(`${INSURANCE_ROOT}/${referenceNumber}${ALL_SECTIONS}`);
+  } catch (err) {
+    console.error('Error updating application', { err });
```
Comment (Zainzzkk) [inline src/ui/server/controllers/insurance/policy-and-export/about-goods-or-services/index.ts:114]: is it worth adding which section we were in to the console.error so it's easier to track where it failed if looking at logs?

File: e2e-tests/content-strings/error-messages.js (hunk 1 above)
Comment (ttbarnes) [reply e2e-tests/content-strings/error-messages.js:147]: Great spot

**Review by Zainzzkk — APPROVED** [review]
LGTM

**Pull request merged**